cocotb/decorators.py

```diff
@@ -128,7 +134,50 @@ def __iter__(self):
         return self
 
     def __str__(self):
-        return str(self.__qualname__)
+        return "<{}>".format(self.__name__)
+
+    def _get_coro_stack(self):
+        """
+        Get the coroutine callstack of this Task.
+        """
```

Comment:
```suggestion
        """Get the coroutine callstack of this Task."""
```
https://www.python.org/dev/peps/pep-0257/#one-line-docstrings (because I don't have to say anything substantial :)